code/API_definitions/check-device-connectivity.yaml

```diff
@@ -79,6 +79,46 @@ paths:
           $ref: "#/components/responses/Generic500"
         '503':
           $ref: "#/components/responses/Generic503"
+   /connectivity-status:
+    post:
+      tags:
+        - Check Device Connection
+      summary: 'Check if the device is currently connected to the network'
+      operationId: checkConnectivityStatus
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/RequestConnectivityStatus'
```

Comment:
![image](https://user-images.githubusercontent.com/112983658/226058224-ed052f23-2847-4d66-94cb-a3b61214f952.png)

You are requesting just for one device.
It needs to be a list.